.travis.yml

```diff
@@ -93,7 +93,16 @@ jobs:
       script:
         - pytest -vv -rA --diff-type=split tests/acceptance/test_acceptance/ --host http://localhost:8080
 
-    - stage: Build
+    - stage: Test Build using latest tag (no upload)
+      name: linux
+      os: linux
+      dist: xenial
+
+      script:
+        - export APP_VERSION=$(git describe --abbrev=0 --tags | tr -d '^v')
+        - $TRAVIS_BUILD_DIR/scripts/ci_create_packages.sh
+
+    - stage: Build and Upload
       if: type = push AND tag IS present
```

Comment:
added a new stage to test building docker images without uploading them... less clunky debugging

Reply:
I will recommend to use `wait-for-it` to check port is opened.

Reply:
agent is not being run, its just built and packaged so there's no port involved here